taworks/taform/models.py

```diff
@@ -150,6 +150,13 @@ class Meta:
         model = Application
         fields = ['instructor_preference']
 
-class Application_status(models.Model):
+class ApplicationStatus(models.Model):
     status_date = models.DateTimeField(auto_now_add=True)
     status = models.BooleanField(default=False, blank=True)
+
+class Assignment(models.Model):
+    created_at = models.DateTimeField(auto_now_add=True)
+    course_id = models.CharField(max_length=1000, null=True)
```

Comment:
course_id should be
course_id = models.ForeignKey(Course, on_delete=models.CASCADE)
and similarly student_id should be
student_id =
course_id = models.ForeignKey(Student, on_delete=models.CASCADE)

so when new courses are uploaded this table gets wiped too, also the null constraints are taken care of, we do not want nulls for those 2 columns.